prov/gni/src/gnix_cm.c

```diff
@@ -360,6 +363,12 @@ DIRECT_FN STATIC int gnix_connect(struct fid_ep *ep, const void *addr,
 	req.cq_irq_mdh = ep_priv->nic->irq_mem_hndl;
 	req.peer_caps = ep_priv->caps;
 
+	req.cm_data_len = paramlen;
+	if (paramlen) {
```

Comment:
should there be a check for paramlen being too large?  or is this checked somewhere else?